openssl-dynamic/src/main/c/ssl.c

```diff
@@ -54,6 +56,17 @@ struct CRYPTO_dynlock_value {
     apr_thread_mutex_t *mutex;
 };
 
+struct TCN_bio_bytebuffer {
```

Comment:
I think I will leave it here for now unless you are strong on this. It is only used in the context of `ssl` and that is how the existing code is generally partitioned.